travis-ci/vmtest/run_selftests.sh

```diff
@@ -4,27 +4,41 @@ set -euo pipefail
 
 source $(cd $(dirname $0) && pwd)/helpers.sh
 
+STATUS_FILE=/exitstatus
+
 test_progs() {
 	if [[ "${KERNEL}" != '4.9.0' ]]; then
 		travis_fold start test_progs "Testing test_progs"
+		set +e
```

Comment:
you don't really need to set/reset +e, you should be good with adding `|| true` to prevent the shell from erroring out

Reply:
But that wouldn't allow me to get the correct return code with `$?` on the following line, I believe?

Reply:
It should be `&& true`, sorry. We do that already: https://github.com/kernel-patches/vmtest/blob/master/travis-ci/vmtest/run.sh#L385-L389

Reply:
Right, should have remembered that :facepalm:. The `||` put me off. Ok will do, it will be more readable.